README.md

```diff
@@ -11,18 +11,16 @@
 <a title="CodeCov" href="https://codecov.io/gh/arcana-network/auth"> 
  <img src="https://codecov.io/gh/arcana-network/auth/branch/dev/graph/badge.svg?token=KmdjEs3enL"/></a>
 </p><p id="start" align="center">
-<a href="https://docs.beta.arcana.network/"><img src="https://raw.githubusercontent.com/arcana-network/branding/main/an_banner_docs.png" alt="Arcana Auth SDK"/></a>
+<a href="https://docs.beta.arcana.network/"><img src="https://raw.githubusercontent.com/arcana-network/branding/main/an_auth_sdk_banner_feb_24.png" alt="Arcana Auth SDK"/></a>
 </p>
 
-# Arcana Auth
+# Auth
 
-Use Arcana Auth SDK to onboard Web3 app users via social login and allow authenticated users to instantly access the built-in, embedded Arcana wallet and sign blockchain transactions from within the app context. 
+[Arcana Network](https://arcana.network) Auth SDK integrates with Web3 apps to enable social login and embedded, in-app Arcana wallet. Users don't need to install a wallet. Developers can leverage the built-in gasless feature to sponsor gas fees for white-listed blockchain transactions. 
```

Comment:
Isn't it the other way around? Web3 applications can integrate with the Arcana Network Auth SDK to enable social logins, etc.?